src/components/MoneyRequestConfirmationList.js

```diff
@@ -206,14 +206,17 @@ function MoneyRequestConfirmationList(props) {
     );
 
     /**
-     * Navigate to profile of selected user
+     * Navigate to report details or profile of selected user
      * @param {Object} option
      */
-    const navigateToUserDetail = (option) => {
-        if (!option.accountID) {
+    const navigateToReportOrUserDetail = (option) => {
+        if (option.accountID) {
+            Navigation.navigate(ROUTES.getProfileRoute(option.accountID));
             return;
```

Comment:
is return required here now? I don't think so.

Reply:
Yes, I've removed it.